invenio_communities/communities/records/systemfields/is_verified.py

```diff
@@ -23,11 +24,15 @@ def calculate(self, record):
         # import here due to circular import
         from invenio_communities.members.records.api import Member
 
-        community_verified = False
         owners = [m.dumps() for m in Member.get_members(record.id) if m.role == "owner"]
+
+        # community is considered verified if at least one owner is verified
+        # a user is verified if verified_at is not None
+        # a group is per definition verified, because it could only be created by admin
         for owner in owners:
-            # community is considered verified if at least one owner is verified
-            if owner["user"]["verified_at"] is not None:
-                community_verified = True
-                break
-        return community_verified
+            if "user" in owner and owner["user"]["verified_at"] is not None:
+                return True
+            if "group" in owner:
```

Comment:
We discussed with @kpsherva and @slint that maybe for now is better not to take into account groups as part of the verification mechanism. So, just fixing the "user" key access should be enough.

Reply:
may i asking why is it better?

that would mean that a community is not verified if a group is the owner.

and yes technically i am fixing two problems with this commit.

Reply:
Groups at the moment are not part of "user moderation" which means that potentially a user can trick the system (consider the case of Zenodo where users that are potential spammers) and verify their spam community. 

What is the second problem that you are fixing?

Reply:
trick the system would only work if the user is part of that group and therefore verified already or?

i mean that the commit fixes the KeyError "user" not existing and that groups are not handled. they are independent of each other so this commit fixes two problems which is not fully mentioned in the commit message and also it not really asked in the issue

Reply:
> trick the system would only work if the user is part of that group and therefore verified already or?
Yes you are right about it if you always consider that groups are considered "verified". I would feel probably safer to not take into account the groups as they are not part of any moderation and thus is not a conscious choice. 

An alternative would be to have a configurable choice to indicate "trust or not" groups. Maybe  @slint @kpsherva can also share their thoughts :)

Reply:
I think more prudent approach for now would be not to take groups for granted - I wouldn't mark them as verified by default, by doing that we open a possible "workaround" for verifying the community, but also we are defining a base workflow which might be difficult to change: if we decide to verify groups in any way, the workflow will change and it will be hard to make it backwards compatible if we decide to become less permissive in the verification process in the future - therefore I would also return False here instead